releases/zeppelin-release-0.7.2.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: page
+title: "Apache Zeppelin Release 0.7.2"
+description: ""
+group: release
+---
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+{% include JB/setup %}
+
+## Apache Zeppelin Release 0.7.2
+
+The Apache Zeppelin community is pleased to announce the availability of the 0.7.2 release.
+
+The community put significant effort into improving Apache Zeppelin since the last release.
+25 contributors provided 50+ patches for improvements and bug fixes.
+More than 40+ issues have been resolved.
+
+We encourage to [download](../../download.html) the latest release. Feedback through the [mailing lists](../../community.html) is very welcome.
+
+<br />
+You can visit [issue tracker](https://issues.apache.org/jira/secure/ReleaseNote.jspa?version=12340231&projectId=12316221) for full list of issues have been resolved.
+
+
+<br />
+### Contributors
+
+This release would not be possible without the following community members' contributions:
+
+Jeff Zhang, 1ambda,  Lee moon soo, Prabhjyot Singh, soralee, AhyoungRyu, Khalid Huseynov, Nelson Costa, Benoy Antony, Christian Fries, CloverHearts, Vipin Rathor, LeiWang, Mina Lee, Jongyoul Lee, sravan, Frank Rosner, Paul Brenner, Elek Márton, Renjith Kamath, Sohaib Iftikhar, William Markito Oliveira, andrea, sergeymazin
```

Comment:
I use the name that user set in their git config, I cannot know full name of the author if there is no information in the git log :(